packages/react/src/components/Lightbox/Lightbox.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ExpressiveModal } from '../ExpressiveModal';
+import { ModalBody } from 'carbon-components-react';
+import { settings } from 'carbon-components';
+import { DDS_LIGHTBOX } from '../../internal/FeatureFlags';
+import {
+  featureFlag,
+  settings as ddsSettings,
+} from '@carbon/ibmdotcom-utilities';
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Lightbox Component
+ *
+ * @param {object} props props object
+ * @param {string} props.title LightBox media title
+ * @param {string} props.copy Lightbox media short description
+ * @param {string} props.image Lightbox responsive image object
+ * @param {boolean} props.open sets whether the modal is open/close
+ * @param {Function} props.onClose do something on close in addition, return false to completely replace
+ * @returns {*} JSX Object
+ */
+const Lightbox = ({ title, copy, image, open, onClose }) => {
+  /**
+   * Set default image
+   *
+   * @param {*} image responsive image object
+   * @returns {string} default image url
+   */
+  const defaultImage = image => {
+    return image.uri.md
+      ? image.uri.md
```

Comment:
I think we've been more consistently using `src`, so maybe something like `image.src.md` would be closer. I think this will be more defined once the responsive image component begins taking shape.

Reply:
I think this is one of the things that need refactoring and standardizing. For this alpha pattern , I am using same image object format that I used in use cases. 
@jeffchew @photodow 
I can refactor for both this component and use cases  pattern if we all want it.

Reply:
It's probably not worth refactoring anything until it's a little more concrete.